### PR TITLE
A package for Archlinux

### PR DESCRIPTION
**Pull request by frankenstein91**
I would like to take this opportunity to provide my **first** Linux package. I hope there are no major errors in the PKGBUILD.

If changes are needed, feel free to write me here.
Also, I haven't filled the maintainer yet because I didn't know if you would like to do it yourself.

### CODE REVIEW

**Comment by NaitLee** [comment]
Wow, I'm really moved this time 🥲

I **really** like Arch Linux, (the [Artix](https://artixlinux.org/) derivation is also great for me, many ones and the less-experienced)
I am feeling very freedom here, without the bothering of "release", "brand" or "official".
Everyone have his/her role, the community is **the** best to do whatever!

There's a consideration, should we install the python dep `bleak` via `pip` or just put in the package?
The difference is 100KiB of additional data (zipped, even less if use zst).

Also I'm wondering, after we package it, should we put it to community repo or AUR?
And a common question, is AUR very popular among there, and convenient enough?
I'm not experienced in packaging, I need guidance 😁

**Comment by frankenstein91** [comment]
Hello NaitLee,
I have just taken care of the dependency problem with bleak.

If I have understood the whole thing correctly, you always publish a software package in the AUR first. The packages can then be voted for there, and if there are enough votes, you can then switch to the community repo.

I'm not experienced in packaging as well. But you have to start somewhere.

File: PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+pkgdesc="A project that provides support to some Bluetooth Cat Printer models, on many platforms!"
+arch=('any')
+url="https://github.com/NaitLee/Cat-Printer"
+license=('GPL3')
+depends=('python' 'bluez' 'bluez-utils' 'python-bleak')
+makedepends=('git')
+provides=("cat-printer=${pkgver}")
+source=("$pkgname::git+https://github.com/NaitLee/Cat-Printer.git")
+md5sums=('SKIP')
+sha256sums=('SKIP')
```
Comment (frankenstein91) [inline PKGBUILD:15]: I don't think we need both, but it looks cleaner from my point of view.

**Comment by frankenstein91** [comment]
@NaitLee this looks merged to me as well 
https://github.com/NaitLee/Cat-Printer/blame/main/PKGBUILD

I would close it as pull request

**Pull request closed without merging**